### ISSUE TITLE
Deaf characters can hear events of petrified eye mine finale.

### ISSUE THREAD

**Issue by SirPendrak** [CleverRaven/Cataclysm-DDA#28969]
**Describe the bug**  
As title. Deaf characters canot hear screams of eye, but they can hear the cracking rock when wyrms spawn. (edit. In polish version it was saying about rock cracking, in english it apparently mentions sounds made by wyrms, wathever anyway) 

![Mine eye pedestal event](https://user-images.githubusercontent.com/43266802/54955062-8e003d80-4f4c-11e9-8bb0-22bfb9c7510b.jpg)

**Versions and configuration(please complete the following information):**
 - OS: Windows 10
 - Game Version 0.D-572-gc62f18a (tiles)
 - Build 8621
 - Mods loaded:

Dark Days Ahead
Disable NPC Needs
Filthy Clothing
Mutant NPCs
Beta National Guard Camp
Sleep Deprivation
Z-levels enabled

**Additional context**  
Just a remindeer. This event is still kinda broken, because to trigger it you need to activate pedestal, and you can do it only after picking up eye. Most players will just grab the eye, and leave place without even knowing there are supossed to be enemies. Its a known isue for some years apparently. Maybe some other mechanic is needed, like plant gathering to get the eye from pedestal, instead of spawning item on furniture piece?

**Comment by Bulletfang** [comment]
I've been playing for almost a year now and this is the first time I hear about some event in the petrified eye mine.

**Comment by Night-Pryanik** [comment]
Could you please describe in details which sound in particular your character heard while being deaf?

**Comment by SirPendrak** [comment]
Updated with screenshot as proof. At first i wanted to report the whole event being broken, then i finally discovered how to trigger it, but i did not want to make another issue for things that already were reported 3 years ago, so i found something new to report, as an oportunity to remind something more important xd

**Comment by adamkad1** [comment]
isnt it like you only think you hear those sounds?

**Comment by Night-Pryanik** [comment]
Ok, I got it.
